alumnos/ocf/clase-05/11.js

```diff
@@ -0,0 +1,73 @@
+// 11. En casa tenemos una estantería muy interesante de libros. Cada libro se compone de las siguientes propiedades... 
+
+const estanteria = {
+  libros: [{
+    nombre: 'El caballero oscuro',
+    autor: 'Frank Miller',
+    leido: false
+  },
+  {
+    nombre: 'El mundo amarillo',
+    autor: 'Albert Espinosa',
+    leido: false,
+  },
+  {
+    nombre: 'Harry Potter y el caliz de fuego',
+    autor: 'J.K. Rowling',
+    leido: true,
+  },
+  {
+    nombre: 'El ingenioso hidalgo Don Quijote de la Mancha',
+    autor: 'Miguel de Cervantes',
+    leido: false
+  },
+  {
+    nombre: 'Berserk',
+    autor: 'Kentaro Miura',
+    leido: true
+  },
+  {
+    nombre: 'Iliada',
+    autor: 'Homero',
+    leido: false
+  }],
+  
+  infoLibro() {
+    let libros = [];
+
+    for (const libro of this.libros) {
+      let {nombre, autor, leido} = libro;
+      libros.push(libro);
```

Comment:
Esta variable `libros` creo que te sobra 😅

Reply:
😨 Si la quito deja de funcionar, ya que la estoy usando para hacer push y meter dentro de ella los elementos que estoy recorriendo con el for...of

Reply:
¡Tienes razón!
He eliminado la declaración de la variable y la línea del push y todo funciona igual, así que eso no hacía nada.
¡He vivido engañado! 😅